cdrkit.yaml

```diff
@@ -0,0 +1,108 @@
+package:
+  name: cdrkit
+  version: 1.1.11
+  epoch: 0
+  description: Suite of programs for CD/DVD recording, ISO image creation, and audio CD extraction
+  copyright:
+    - license: GPL-2.0-only
+
+environment:
+  contents:
+    packages:
+      - build-base
+      - busybox
+      - bzip2
+      - bzip2-dev
+      - ca-certificates-bundle
+      - cmake
+      - file
+      - gcc
+      - libcap-dev
+      - libmagic-dev
+      - linux-headers
+      - samurai
+      - zlib-dev
+
+pipeline:
+  - uses: fetch
+    with:
+      expected-sha256: d1c030756ecc182defee9fe885638c1785d35a2c2a297b4604c0e0dcc78e47da
+      uri: https://dev.alpinelinux.org/archive/cdrkit/cdrkit-${{package.version}}.tar.gz
+
+  # https://gitlab.alpinelinux.org/alpine/aports/-/tree/master/community/cdrkit
+  - uses: patch
+    with:
+      series: series
+
+  - name: Disable rcmd, it is security risk and not implemented in musl
+    runs: |
+      sed -i include/xconfig.h.in -e "s/#define HAVE_RCMD 1/#undef HAVE_RCMD/g"
+
+  - runs: |
+      cmake -B build -G Ninja \
```

Comment:
we can use our pipeline here same for install step as well.

Reply:
Nice idea, forgot to replace those!